### ISSUE TITLE
Questions regarding MNIST

### ISSUE THREAD

**Issue by ojmakke** [itayhubara/BinaryNet.pytorch#21]
Hello, 

The last layer for MNIST is Linear, not BinarizeLinear, this will cause the weights to not necessarily be binary, correct?

Also, for the batch normalization layer, its parameters are not binary. Correct?
